spec/slack_notifier_spec.rb

```diff
@@ -0,0 +1,43 @@
+require "spec_helper"
+require "./slack_notifier"
+
+describe SlackNotifier do
+  before :each do
+    stub_request(:any, "https://hooks.slack.com")
+  end
+  let(:notifier) { SlackNotifier.new("https://hooks.slack.com") }
+
+  describe ".new_issue_not_published" do
+    it "sends correct payload to slack" do
+      notifier.new_issue_not_published
+      expect(WebMock).to have_requested(:post, "https://hooks.slack.com").
+                           with(body: '{"text":"*Fíha, dnes na ÚVO nevyšlo nové vydanie vestníka?*","channel":"#general","username":"uvobot","icon_emoji":":mag_right:"}')
```

Comment:
Line is too long. [170/80]<br>Use 2 (not 21) spaces for indenting an expression spanning multiple lines.